Python/Audio_Format_Converter/Audio_File_Converter.py

```diff
@@ -0,0 +1,45 @@
+# Script to Convert audio files from one format to another
+import os
+from glob import glob
+import pydub
+import mutagen
+
+# Directory containing the audio File
+song_dir = 'C:\\Users\\asus\\Desktop\\Songs\\*.mp3'  # Extension is according the the file type before conversion.
```

Comment:
Remove any Local References in File Paths as per your Local Machine.

Reply:
Updated in the latest commit